pages/glossary/distinguishable-styles.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Distinguishable Styles
+key: distinguishable-styles
+unambiguous: true
+objective: true
+---
+
+Two elements have _distinguishable styles_ if at least one of the following is true:
```

Comment:
Should text shadows be allowed?

Reply:
I've updated the text-style-properties to include it